examples/helloworld/main.go

```diff
@@ -30,6 +30,8 @@ func main() {
 	pflag.CommandLine.SetNormalizeFunc(utilflag.WordSepNormalizeFunc)
 	pflag.CommandLine.AddGoFlagSet(goflag.CommandLine)
 
+	logs.NewOptions().AddFlags(pflag.CommandLine)
+
 	logs.InitLogs()
 	defer logs.FlushLogs()
```

Comment:
I think the  `logs.InitLogs()
	defer logs.FlushLogs()` are not necessary if we use `logs.NewOptions().AddFlags(pflag.CommandLine)`

Reply:
done